src/index.ts

```diff
@@ -89,75 +95,82 @@ export default class Printd {
    * @param scripts Optional scripts (script texts or urls) that will add to iframe document.body
    * @param callback Optional callback that will be triggered when content is ready to print
    */
-  print (el: HTMLElement, styles?: string[], scripts?: string[], callback?: PrintdCallback) {
-    if (this.isLoading) return
+  print(
+    el: HTMLElement,
+    styles?: string[],
+    scripts?: string[],
+    callback?: PrintdCallback
+  ) {
+    if (this.isLoading) return;
 
-    const { contentDocument, contentWindow } = this.iframe
+    const { contentDocument, contentWindow } = this.iframe;
```

Comment:
**Fourth**, I have mentioned in the third review above. Here I see a bug when `this.iframe` will be `null` in your use case.
Please ensure and cover this `null` behavior with unit tests.

Reply:
brother .. i m using "  ? : "  this java script if condition .. method . and its perfectly worked for me ..

in gatsby client side building time .. window will not defined so there is the problem .. then i use that if condition if not defined then null ..when window is there then its will work like ur code ..

Reply:
> **Finally**, and not less important.
> Please provide more context and info about the issue that you have found and about the solution that you propose.
> 
> Together with any information that you consider relevant. Screenshots, outputs, demos, etc.
> Otherwise I couldn't understand what this PR is really intended for.